projects/event-lambdas/src/event-api-lambda/__tests__/api-lambda.spec.ts

```diff
@@ -37,7 +37,7 @@ describe("Event API lambda", () => {
   beforeAll(async () => {
     try {
       await s3.listObjects({ Bucket: telemetryBucketName }).promise();
-    } catch (e) {
+    } catch (e: any) {
```

Comment:
new versions of typescript means we have to explicitly set a type of either `any` or `unknown` here, i set it to `any` as these only ran in tests